cmd/generate/config/rules/octopusdeploy.go

```diff
@@ -0,0 +1,32 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func OctopusDeployApiKey() *config.Rule {
+	// define rule
+	r := config.Rule{
+		RuleID:      "octopus-deploy-api-key",
+		Description: "Discovered a potential Octopus Deploy API key, risking application deployments and operational security.",
+		Regex:       utils.GenerateUniqueTokenRegex(`API-[A-Z0-9]{26}`, false),
+		Entropy:     3,
+		Keywords:    []string{"api-"},
```

Comment:
@rgmz Should `octopus` be a keyword as well?

Reply:
I'm undecided on that. Using `API-` as the keyword could either be high confidence (more so than "octopus"), or result in false positives.

I don't know of any other providers that have the specific format `API-[A-Z0-9]{26}`.

Reply:
👍🏻 lets just keep an eye on this rule